apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-tabs.tsx

```diff
@@ -171,7 +175,8 @@ export const InAppTabs = ({ workflow, step }: { workflow: WorkflowResponseDto; s
     [formValues]
   );
 
-  const blocker = useBlocker(formState.isDirty || isPending);
+  const isNotDirty = Object.keys(formState.dirtyFields).length === 0;
+  const blocker = useBlocker(isNotDirty || isPending);
```

Comment:
This blocker stops me from closing the drawer, because the form is not dirty on the first load.

https://github.com/user-attachments/assets/0f7702f5-110e-4653-b758-54e57447e4b5

Reply:
ok yes, should be `!isNotDirty`